src/components/visualizations/chart/highcharts/test/getOptionalStackingConfiguration.spec.ts

```diff
@@ -639,6 +639,23 @@ describe("getOptionalStackingConfiguration", () => {
             expect(result.yAxis[0]).toHaveProperty("labels.formatter");
             expect(result.yAxis[1]).toEqual({});
         });
+
+        it("should NOT add formatter when primary y-axis is line chart type in combo chart", () => {
+            const chartOptions = {
+                type: VisualizationTypes.COMBO,
+                yAxes: [{ opposite: false }],
+                data: {
+                    series: Array(2).fill({ yAxis: 0 }),
+                },
+            };
+            const chartConfig = {
+                stackMeasuresToPercent: true,
+                primaryChartType: VisualizationTypes.LINE,
+            };
+
+            const result: any = getShowInPercentConfiguration(chartOptions, undefined, chartConfig);
```

Comment:
it's `any` by default, so ti could be `const result = ...`

Reply:
fixed